tests/test_helpers.py

```diff
@@ -158,6 +158,12 @@
      "2020-05-27: lead: ascopatz, sb_firstprojectum, status: proposed\n    Type: mergedpr\n    Title: planning meeting\n    log url: https://docs.google.com/document/d/1YC_wtW5Q\n    Purpose: develop a detailed plan with dates\n    Audience: ascopatz, scopatz, ascopatz\n"
      "2020-05-20: lead: ascopatz, sb_firstprojectum, status: proposed\n    Type: meeting\n    Title: Project lead presentation\n    log url: https://docs.google.com/document/d/1YC_wtW5Q\n    Purpose: lead presents background reading and initial project plan\n    Audience: ascopatz, scopatz, ascopatz\n    Notes:\n      - do background reading\n      - understand math\n"
      ),
+    (["helper", "l_milestones", "--lead", "pliu"],
```

Comment:
these look like new tests (since I don't see any lines that are in red that will be deleted).  Since you are updating an existing helper there will almost certainly (should) be existing tests.  So your job is to go and find all the `l_milestones` tests and _update_ them so they exhibit the desired behavior.  You may also want/need to add additional new tests if there is new functionality that needs to be tested, but in this case we are not changing functionality, only what we want printed on the output line, so I guess we won't need any new tests, just updated existing tests.

Reply:
Yes. I wrote two new tests to get a better understanding of the testing structure. I will get rid of these and update the existing tests. 

Also, I have a rookie question about my `issue_#985` branch. My main is synchronized as far as I can see, but how can I tell if the branch of my PR is synced to the latest version of the main branch? I am also having an issue with my gitlab login. I will follow up with you on Slack about that.